wordpress/deployer/Dockerfile

```diff
@@ -13,8 +14,7 @@ RUN cat /tmp/schema.yaml \
     > /tmp/schema.yaml.new \
     && mv /tmp/schema.yaml.new /tmp/schema.yaml
 
-
-FROM gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst
+FROM gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:$MARKETPLACE_TOOLS_TAG
```

Comment:
This is gonna add to confusing readers of this example repo!

We have this problem with too much magic and convention in the example repo. I imagine the example repo will play less of an important role, now that we have more thought out guides, so it's not too critical to go and fix all of these issues.

Reply:
Agreed - magic is bad.

In my opinion, these examples are too involved - I think we could make them more useful by having them start bare and have guides to add features.